Improved documentation for missing public directory (#1020)
zeit/docs: pages/docs/v2/advanced/platform/frequently-asked-questions.mdx

## Missing Public Directory

> If your project contains a `package.json` file with a `build` [script](https://docs.npmjs.com/misc/scripts), it is expected dump its static output files into a directory named `public` when the `build` [script](https://docs.npmjs.com/misc/scripts) is executed.

Reviewer: ```suggestion
If your project contains a `package.json` file with a `build` [script](https://docs.npmjs.com/misc/scripts), it is expected to output its static files into a directory named `public`, at the root of the project, when the `build` [script](https://docs.npmjs.com/misc/scripts) is executed.
```

> You are seeing this error because your setup didn't satisfy this contraint.

Reviewer: ```suggestion
You are seeing this error because your setup did not satisfy the above constraint.
```

> To solve the problem, configure your project to place its static output files into a `public` directory when the `build` [script](https://docs.npmjs.com/misc/scripts) is invoked.

Reviewer: ```suggestion
To resolve this error, configure your project to place its static output files into a `public` directory, at the root of your project, when the `build` [script](https://docs.npmjs.com/misc/scripts) from the `package.json` is invoked.
```

> For example, this is how your `package.json` file might look like:

Reviewer: ```suggestion
For example, this is how your `package.json` file might look:
```

```json
{
  "scripts": {
    "build": "my-framework build"
  }
}
```

> If you are seeing this error, you need to adjust the command to direct its static output files into the `public` directory instead. Depending on your framework, the option for this might be called `--output`, `--out`, `--output-dir` or something similar:

Reviewer: ```suggestion
If you are seeing this error with a similar `build` script already, you may need to adjust the command to direct its static output files into the `public` directory instead.

Depending on your framework, the option for this might be called `--output`, `--out`, `--output-dir` or similar:
```

```json
{
  "scripts": {
    "build": "my-framework build --output public"
  }
}
```

> Once you have done this, the error will disappear because the output will end up in the right place.

Reviewer: ```suggestion
Once you have done this, the error will disappear since the output will be the expected directory, `public`.
```
